app/config/parameters.yml.dist

```diff
@@ -73,3 +73,6 @@ parameters:
 
     # pocket import
     pocket_consumer_key: xxxxxxxx
+    
+    # sites credentials
+    sites_credentials: {}
```

Comment:
Or we abstract credentials retrieval, with a first implementation that uses a config variable. We can add a new, multi-user / database based one at any time. What do you think ? 

To abstract it, I'd add a SitesCredentialsProvider, that replaces the array. What do you think ?